bin/buildIcons.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const globby = require('globby');
+const path = require('path');
+const iconFolder = path.resolve(__dirname, '../src/helix-ui/icons/')
+
+const Icons = {};
+globby.sync('*.svg', { cwd: iconFolder }).forEach(file => {
+    let filename = path.join(iconFolder, file)
+    let data = fs.readFileSync(filename, { encoding: 'utf8' });
+    Icons[file] = data;
```

Comment:
`file` should be the basename of the svg file. i.e. `account.svg` should set `Icons['account']`, etc.